src/porepy/__init__.py

```diff
@@ -110,6 +110,7 @@
 )
 
 # Fractures
+from porepy.fracs.fracture import Fracture
```

Comment:
Why do we need to include the `Fracture` class in the top-level init? This is an abstract class that _should_ never be instantiated. Right?

Reply:
I had a reason for it, but I think you're right.